tests/test_config.py

```diff
@@ -1017,7 +1018,7 @@ def test_semip_config(self):
         self.assertTrue(model.dim == 2)
         self.assertTrue(model.inducing_size == 10)
         self.assertTrue(model.stim_dim == 1)
-        self.assertTrue(model.inducing_point_method == "sobol")
+        self.assertTrue(model.inducing_point_method == SobolAllocator)
```

Comment:
Let's make a new file for InducingPointAllocator tests and check that each class can be correctly read from a config with its arguments (if any).